src/backofftimer.cpp

```diff
@@ -136,4 +136,85 @@ TimerWithBackoff::TimerWithBackoff(PrnGen &rng, int tag)
     this->tag = tag;
 }
 
+
+void BackoffTimerGroupTracker::update(dstime* waituntil, bool transfers)
+{
+    vector<BackoffTimerTracked*> v;
```

Comment:
Perhaps calling it `expiredTimers` or `armedBackoffs` could improve readability :)

Reply:
I think we should keep calling this one `udpate`, since it's doing the same thing as BackoffTimer::update, but for the whole group - I've added comments to explain that aspect.  thanks

Reply:
Oh, I meant the short-named variable `v`, not the method itself :rofl: 
Thanks for the explanatory comments, anyway.